NewForce-Cohort-9/y-all-hungry-mountain-momma-meat-sticks
# Tyler branch

# Description

Revamps to foods.js, desserts.js, and drinks.js, as well as adding a new file that helps provide a running total of a customers order.  Also changed main around to ensure it renders.

# Known issues

When hitting "place order" the order placed does NOT retain the knowledge of what was placed, and it continually sets the value of everything to 0, causing every new order to look like it is for nothing.

# How to test?

Run my code, play around with the food options, and verify that the running counter is now going as you change options.
